src/components/mainScreen/mainScreen.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
```

Comment:
make an index file in every folder and export the component from there, so we can simply import from index.